mozilla/ichnaea
# centralize database uri for alembic migrations

There are a few ways we run alembic migrations:

1. running the alembic command from the shell to run migrations
   in a local dev environment
2. running the alembic command from the shell to run migrations
   in a server environment
3. running alembic subcommands from ichnaea/scripts/db.py
4. running alembic subcommands for test setup

We want all of these cases to pull the database uri from an environment
variable. This changes the code such that all of these use cases use the
`SQLALCHEMY_URL` environment variable.

In the local dev environment, this value is in `docker/config/app.env`. In a server environment, they'll be provided on the admin node in the environment.

To test:

1. run `make build` and `make setup`
2. run `make test`
3. run `make shell` and then at the prompt `alembic current -v`

All of those situations should work.

Fix #904.
